commands/faas.go

```diff
@@ -53,8 +54,10 @@ func Execute(customArgs []string) {
 	checkAndSetDefaultYaml()
 
 	faasCmd.SilenceUsage = true
+	faasCmd.SilenceErrors = true
 	faasCmd.SetArgs(customArgs[1:])
 	if err := faasCmd.Execute(); err != nil {
+		fmt.Println(capitaliseFirst(err.Error()))
```

Comment:
Let's just inline the code for capitaliseFirst.

Is there an issue with something simpler like this?

```
error := "lowercase error"
fmt.Println(strings.ToUpper(error[:1]) + error[1:])
```

Is this because you were expecting multi-byte characters?

Reply:
There was a case where we returned a 0 length error message because another function was printing errors as well. I refactored the same code, so now all error messages should be longer that 0 chars long, making it fairly safe to use the method you mentioned here.

I hadn't really considered multibyte characters, there are a few error messages that starts with user input, so I tested those.

capitaliseString would look like this:
```
$ faas-cli new test --lang ルビ
ル��ビ is unavailable or not supported
```
while using your solution works as expected
```
$ faas-cli new test --lang ルビ
ルビ is unavailable or not supported
```
I tried some arabic as well as chinese characters as well, all seems to work, but I got to admit I don't know enough about these issues to be sure.
```
$ faas-cli new test --lang 鬱    
鬱 is unavailable or not supported
$ faas-cli new test --lang وتري  
وتري is unavailable or not supported
```